### DISCUSSION TITLE
Modeling open water bodies

### DISCUSSION THREAD

**Discussion by julianseb** [idaholab/moose#29528] [Q&A Modules: Porous Flow]
Hi everyone,

i'm doing a model where i have groundwater flow with an open water body an they are in an exchange. Right now i try to model the open water body within PorousFlow and the necessary materials. But im running into Problems with my pore- and gaspressures while trying to implement two phase flow in the ground with seeing the water body as fully saturated and i'm wondering if it maybe because of the materials i set for the water or because i have unrealistic BCs. Because i'm not sure how to define a realistic permeability for the waterbody. Right now i have porosity set to 0.9999 and permeability at 1E-6. 

Are there any "good ways" to model an open water body in PorousFlow? Are there alternatives from another module?

Thanks in advance

**Comment by GiudGiud** [comment]
@cpgr @WilkAndy

**Comment by josebastiase** [comment]
Hi,
Can you provide more information about what are you trying to do? Maybe a simple example if what you have so far may help.

**Reply by julianseb** [reply]
Hi, 
of course. I'm trying to model the thermal influence of an underground power cable on groundwater and open water bodies. My conceptual model looks like this:

![Model](https://github.com/user-attachments/assets/fcf7486d-4e6e-4404-8864-647364a90283)

I'm having a gradient from left to right. Right now i'm trying to implement two phase flow in the soil while seeing the water body as fully saturated. I'm running into problems with my water pressure on initialization. I'm using Water97FluidProperties but i get negative pressure values so the initilazation aborts. I'm not sure if it is because i use different dictators for the water body and the soil, my BCs or ICs or what exactly is the reason so i wondered if there maybe is another perhaps better or easier way to implement the water body.

I also added my current state.


[Current_state.zip](https://github.com/user-attachments/files/18125292/Current_state.zip)

**Reply by josebastiase** [reply]
I’ve never tried modeling something like this myself, but it does sound very interesting! If I were in your shoes, here’s how I’d approach debugging and simplifying the setup:

Start by removing the heat physics (temporarily) to focus only on the groundwater flow and saturation behavior. This might help you pinpoint whether the problem lies in the two-phase flow implementation or elsewhere.

Explore using MOOSE Transfer module and variable transfers at the boundaries between the water body and the soil.

Sometimes in multiphase simulations, where a phase has saturation zero wont converge in FE, try a FV implementation.

sorry I cant help,
